regression-test/src/app/components/[componentName]/page.tsx

```diff
@@ -21,15 +21,22 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import { ApplyTextDirection } from '@instructure/ui'
-
+'use client'
 import React from 'react'
-const APPLYTEXTDIRECTION_IUI = () => {
+import { View } from '@instructure/ui'
+
+export default function Component({ params }: any) {
+  const Component = require(`../../components/${params.componentName}`)
+    .default as React.ElementType
   return (
```

Comment:
should we wrap this in InstuiSettingsProvider to more accurately simulate realworld usage?

Reply:
I added it.